app/components/Table.tsx

```diff
@@ -74,6 +74,7 @@ interface FilterItem {
 }
 
 interface Filter {
+  key?: string;
```

Comment:
is the key optional? Or are there cases where it's not needed?

Reply:
it is optional so that, will include the test ids only exists.

Reply:
ah cool